users/forms.py

```diff
@@ -46,8 +50,8 @@ class Meta:
 class ProfileUpdateForm(forms.ModelForm):
     experience = forms.IntegerField()
     date_of_birth = forms.DateField(widget=DateInput())
-    education = forms.ChoiceField(choices=EDUCATION_CHOICES)
-    gender = forms.ChoiceField(choices=GENDER_CHOICES)
+    education = forms.ChoiceField(choices=EDUCATION_CHOICES_EMPTY_LABEL)
+    gender = forms.ChoiceField(choices=GENDER_CHOICES_EMPTY_LABEL)
```

Comment:
Probably it might be more useful to separate different versions of "choices" between field and widget. E.g. something like next one:
```python
education = forms.ChoiceField(
    choices=EDUCATION_CHOICES, 
    widget=forms.SelectWidget(choices=EDUCATION_CHOICES_EMPTY_LABEL)
)
```